package.json

```diff
@@ -81,12 +81,20 @@
                         "internalDebuggerPort": 9090,
                         "sourceMaps": true,
                         "outDir": "${workspaceRoot}/.vscode/.react"
+                    },
+                    {
+                        "name": "Attach to packager",
+                        "program": "${workspaceRoot}/.vscode/launchReactNative.js",
+                        "type": "reactnative",
+                        "request": "launch",
+                        "internalDebuggerPort": 9090,
+                        "sourceMaps": true,
+                        "outDir": "${workspaceRoot}/.vscode/.react"
                     }
                 ],
                 "configurationAttributes": {
                     "launch": {
                         "required": [
-                            "platform",
```

Comment:
I tried using attach (instead of launch), but the project didn't build. Is there a set of rules somewhere I need to change to be able to use attach?